src/docs/cookbook-manage/implement-delete-user/index.md

```diff
@@ -0,0 +1,574 @@
+manually for deleted users, if the transactions have otherwise been
+completed.
+
+## Add UI components
+
+### Add "Delete account" tab to Account settings
+
+First, add a new tab to the Account settings section of the template.
+
+```shell
+└── src
+    └── components
+         └── LayoutWrapperAccountSettingsSideNav
+              └── LayoutWrapperAccountSettingsSideNav.js
+          ...
+```
+
+The new page will appear as the final tab in Account Settings side
+navigation.
+
+```diff
+  const tabs = [
+    {
+      text: <FormattedMessage id="LayoutWrapperAccountSettingsSideNav.contactDetailsTabTitle" />,
+  ...
+      linkProps: {
+        name: 'PaymentMethodsPage',
+      },
+    },
++   {
++     text: <FormattedMessage id="LayoutWrapperAccountSettingsSideNav.deleteAccountTabTitle" />,
++     selected: currentTab === 'DeleteAccountPage',
++     id: 'DeleteAccountPageTab',
++     linkProps: {
++       name: 'DeleteAccountPage',
++     },
++   },
+  ];
+```
+
+Next, add the actual DeleteAccountPage component. You can use the
+example files directly, or modify them for your use case.
+
+```shell
+└── src
+    └── containers
+         └── DeleteAccountPage
+              └── DeleteAccountPage.js
+              └── DeleteAccountPage.duck.js
+              └── DeleteAccountPage.module.css
+```
+
+- [DeleteAccountPage.js](/cookbook-assets/DeleteAccountPage.js)
+- [DeleteAccountPage.duck.js](/cookbook-assets/DeleteAccountPage.duck.js)
+- [DeleteAccountPage.module.css](/cookbook-assets/DeleteAccountPage.module.css)
+
+Once the DeleteAccountPage component and corresponding Redux store DeleteAccountPage.duck.js exist in the project, import the Redux
+file reducer to a combined reducer.
+
+```shell
+└── src
+    └── containers
+         └── reducers.js
+```
+For more info on the Redux structure, you can read about
+[the Ducks modular Redux proposal](https://github.com/erikras/ducks-modular-redux),
+which is the proposal of Redux usage followed in the FTW templates.
+
+```diff
+  import ContactDetailsPage from './ContactDetailsPage/ContactDetailsPage.duck';
++ import DeleteAccountPage from './DeleteAccountPage/DeleteAccountPage.duck';
+  import EditListingPage from './EditListingPage/EditListingPage.duck';
+  ...
+
+  export {
+    CheckoutPage,
+    ContactDetailsPage,
++   DeleteAccountPage,
+    EditListingPage,
+```
+
+Since DeleteAccountPage has a new route, add the component to the
+application's route configuration.
+
+```shell
+└── src
+    └── routeConfiguration.js
+```
+
+The "Account settings" side navigation tabs are defined in
+routeConfiguration.js as well, so add DeleteAccountPage in the tab
+array.
+
+```diff
+  const ContactDetailsPage = loadable(() => import(/* webpackChunkName: "ContactDetailsPage" */ './containers/ContactDetailsPage/ContactDetailsPage'));
++ const DeleteAccountPage = loadable(() => import(/* webpackChunkName: "DeleteAccountPage" */ './containers/DeleteAccountPage/DeleteAccountPage'));
```

Comment:
I feel like looking into how we could make the `+` symbols unselectable in these code blocks  😆

Reply:
😂 yep, that would improve UX by a lot 💪